src/util/config.cpp

```diff
@@ -230,43 +232,54 @@ bool init(const std::string& wakeroot_path) {
   {
     auto disallowed_keys = find_disallowed_keys(wakeroot_json, wakeroot_allowed_keys);
     for (const auto& key : disallowed_keys) {
-      std::stringstream s;
-      s << "Key '" << key << "' may not be set in .wakeroot";
+      std::cerr << wakeroot_path << ": ";
+      std::cerr << "Key '" << key << "' may not be set in .wakeroot";
       if (user_config_allowed_keys.count(key) > 0) {
-        s << " but it may be set in user config";
+        std::cerr << " but it may be set in user config";
       }
-      s << ".";
-      reporter->reportWarning(Location(wakeroot_path.c_str()), s.str());
+      std::cerr << "." << std::endl;
+      ;
```

Comment:
using `reportWarning` was an explicit choice so that our warnings were more consistent (asked about this in a DM and that's what we landed on). Don't mind that its changing, but we should consider that decision/ensure the warning is consistent with others

Reply:
Right but I can't do that inside of the runtime because I don't have a reporter defined.